src/Service/Resolvers/SqlMutationEngine.cs

```diff
@@ -528,6 +537,15 @@ private static OkObjectResult OkMutationResponse(JsonElement jsonResult)
                     (dbResultSet.Rows.FirstOrDefault() ?? new DbResultSetRow()) : null;
                 if (dbResultSetRow is not null && dbResultSetRow.Columns.Count == 0)
                 {
+                    if (operationType is Config.Operation.Create)
```

Comment:
Can't this operation be Operation.Insert?

Reply:
I don't see Operation.Insert being used for gql anywhere..